keyboards/8pack/rev12/config.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+#include QMK_KEYBOARD_CONFIG_H
```

Comment:
This isn't actually doesn't do anything.  It should be removed

```suggestion
```

Reply:
Done